### PR TITLE
F25 devel rhbz 1321288

### PR DESCRIPTION
**Pull request by rvykydal**
Port from rhel7-branch.

### CODE REVIEW

File: pyanaconda/network.py [hunk 1]
```diff
@@ -686,7 +686,7 @@ def ksdata_from_ifcfg(devname, uuid=None):
     if devname in nm.nm_devices():
         # virtual devices (bond, vlan, ...) not activated in installer
         # are not created so guard these checks
-        if nm.nm_device_is_slave(devname):
+        if nm.nm_device_is_slave(devname) and nm.nm_device_type_is_ethernet(devname):
```
Comment (vpodzime) [inline pyanaconda/network.py]: Doesn't this require a change of the comment right above?

**Comment by vpodzime** [comment]
Looks good to me otherwise.

File: pyanaconda/network.py (hunk 1 above)
Comment (rvykydal) [inline pyanaconda/network.py]: The comment actually talks about the guard above (if devname in nm.nm_devices()), the changed line is just one of "these checks". (The change is that we want to ingore slaves only if they are ethernet devices, ie not bond devices which - unlike ethernet slaves - should have its own kickstart network command)
Anyway, the comment should be moved above the guard line so it is clear what it is talking about?

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
jenkins, test this please

File: pyanaconda/network.py (hunk 1 above)
Comment (vpodzime) [inline pyanaconda/network.py]: Yeah I think it the comment would make more sense if moved above the guard. And maybe we need another comment for the added check?

**Pull request merged**